lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -1014,6 +1024,8 @@ def build_new_order_xml(action, money, payment_source, parameters = {})
             add_stored_credentials(xml, parameters)
             add_pymt_brand_program_code(xml, payment_source, three_d_secure)
             add_mastercard_fields(xml, payment_source, parameters, three_d_secure) if mastercard?(payment_source)
+            add_third_party_token(xml, parameters) if parameters[:store]
+            add_token_transaction_type(xml, parameters) if parameters[:transaction_token]
```

Comment:
Is it possible to have parameters with `:store` and `:transaction_token`? 

What happens in TokenTxnType in that case?

Reply:
the `store` would be a GSF to indicate that we are going to prcoess with a store, for Orbita is set `TokenTxnType` to 'GT'
the `transaction_token` is to send the token stored.

Reply:
What happens if both values come in the parameters hash?